documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -170,18 +178,24 @@ spec:
   map:
     - destination:
         storageClass: <storage_class>
+        accessMode: ReadWriteOnce
       source:
-        id: <source_datastore_mor> <1>
+        id: <source_datastore> <1>
+    - destination:
+        storageClass: <storage_class>
+        accessMode: ReadWriteOnce
```

Comment:
@istein1  Is `ReadWriteOnce` mandatory or can the user choose other access modes?